src/shared/calculations.js

```diff
@@ -9,46 +10,132 @@ import quantile from "@stdlib/stats/base/dists/beta/quantile";
-     *     first element represents the fraction of all ratings which were 1 star, the
-     *     second element represents the fraction of all ratings which were 2 stars,
-     *     and so on.
-     * @param {number} numRatings - total number of ratings given.
+     * @param {number} avgRating - average value of all star ratings
+     * @returns {number} proportion - proportion of ratings which were positive
+     */
+    getProportionPositiveFromAvg(avgRating) {
+        const proportion = (avgRating - 1) / 4;
+        return proportion;
+    },
+
+    /**
+     * By finding the number of ratings which were 4 or 5 stars, compute the proportion 
+     * of reviews which gave a product a "positive" rating. Since Amazon provides a 
+     * histogram showing the percentage of reviews which fell in each star category, we 
+     * just need to sum the 4 and 5 star percentages.
      *
-     * @returns {ConfidenceInterval}
+     * @param {number[]} ratingDistribution - 5-element array, where the first element 
+     *     represents the fraction of all ratings which were 1 star, the second element 
+     *     represents the fraction of all ratings which were 2 stars, and so on.
+     * @returns {number} proportion - proportion of ratings which were positive
      */
-    evaluateRatings(ratingDistribution, numRatings) {
-        const fracPositive = ratingDistribution[3] + ratingDistribution[4];
-        const numPositive = Math.floor(fracPositive * numRatings);
-        return getBetaConfidenceInterval(numRatings, numPositive, 0.95);
+    getProportionPositiveFromDist(ratingDistribution) {
+        const proportion = ratingDistribution[3] + ratingDistribution[4];
+        return proportion;
     },
 
     /**
-     * Get a "positive experience rating" confidence interval based only on the
-     * average star rating and the total number of ratings. Scale ratings in the
-     * domain [1, 5] to the range [0, 1], then treat the result like the average
-     * success rate of a series of independent Bernoulli trials with fixed (but
-     * unknown) probability of success.
+     * Compute the proportion of reviews which gave a product a "positive" rating,
+     * either using the "avg" method (where the input is the average star rating) 
+     * or "dist" method (where the input is the 5-element discrete distribution
+     * from the Amazon rating percentage histogram).
+     *
+     * @param {Object.<string, (number|number[])>} ratingData - object containing
+     *     the average rating under key "avg" and/or the 5-element rating
+     *     distribution (fraction of ratings at each star value) under key "dist".
+     * @param {string} method - which method to use to compute proportion of
+     *     reviews which were positive; must be either "avg" or "dist".
+     * @returns {number} proportion - proportion of ratings which were positive.
+     */
+    getProportionPositive(ratingData, method = "avg") {
```

Comment:
I'm curious why this was introduced as an alternative to directly calling the underlying methods.